modules/obraSocial/routes/obraSocial.ts

```diff
@@ -90,7 +93,12 @@ router.get('/puco', Auth.authenticate(), async (req, res, next) => {
             try {
                 for (let i = 0; i < rta.length; i++) {
                     unaOS = await ObraSocial.find({ codigoPuco: rta[i].codigoOS });
-                    resultOS[i] = { tipoDocumento: rta[i].tipoDoc, dni: rta[i].dni, transmite: rta[i].transmite, nombre: rta[i].nombre, codigoFinanciador: rta[i].codigoOS, idFinanciador: unaOS[0]._id, financiador: unaOS[0].nombre, version: rta[i].version };
+                    if (unaOS?.length) {
+                        resultOS[i] = { tipoDocumento: rta[i].tipoDoc, dni: rta[i].dni, transmite: rta[i].transmite, nombre: rta[i].nombre, codigoFinanciador: rta[i].codigoOS, idFinanciador: unaOS[0]._id, financiador: unaOS[0].nombre, version: rta[i].version };
+                    } else {
+                        // si la OS de puco, no se encuentra en la colección de obrasSociales
+                        obraSocialLog.error('Error falta cargar obra social en su coleccion', { codigoPuco: rta[i].codigoOS }, null, userScheduler);
```

Comment:
el primer argumento es la acción que se ejecuto. Con la función `error` ya estamos indicando que es un error. 

Sería algo mas apropiado algo así:
```suggestion
                        obraSocialLog.error('find', { codigoPuco: rta[i].codigoOS }, null, userScheduler);
```

Reply:
Es verdad, lo corrijo gracias!